pages/common/hydra.md

```diff
@@ -1,6 +1,7 @@
 # hydra
 
 > Online password guessing tool.
+> It supports various protocols: AFP, CVS, FTP, HTTP(S), POP3, RDP, RTSP, SIP, SMB, SMTP, SNMP, SSH, Telnet, VNC, XMPP, ...
```

Comment:
Maybe we can shorten this list to include only the notable ones, like HTTP, POP3, SMTP, SSH, Telnet?

Reply:
I would let it as is ; I already used this tool multiples times in the past and only realized after reading its full help that it supported so many different protocols. As all of the aforementioned protocols are very common, I would not shorten the list if you agree...

Reply:
It seems good this way too. I understand now that other protocols are usable too, and that may be something that users are looking for.
@sbrl what do you say?

Reply:
What about we say something like this?
```suggestion
> Protocols supported include FTP, HTTP(S), SMTP, SNMP, XMPP, SSH, and more.
```

Reply:
OK, done.